components/dashboard/src/components/PrebuildLogs.tsx

```diff
@@ -129,11 +129,13 @@ export default function PrebuildLogs(props: PrebuildLogsProps) {
                     <WorkspaceLogs classes="h-full w-full" logsEmitter={logsEmitter} errorMessage={error?.message} />
                 </Suspense>
             </div>
-            <div className="h-20 px-6 bg-gray-50 dark:bg-gray-800 border-t border-gray-200 dark:border-gray-600 flex space-x-2">
-                {prebuild && <PrebuildStatus prebuild={prebuild} />}
-                <div className="flex-grow" />
-                {props.children}
-            </div>
+            {prebuild && (
+                <div className="h-20 px-6 bg-gray-50 dark:bg-gray-800 border-t border-gray-200 dark:border-gray-600 flex space-x-2 mb-6">
+                    <PrebuildStatus prebuild={prebuild} />
+                    <div className="flex-grow" />
+                    {props.children}
```

Comment:
BTW: What happens if I have a button, say, but no prebuild? :thinking: Not sure it's relevant, but feels like an opportunity to ensure users never get stuck on this view.

Reply:
Related to the [above comment](https://github.com/gitpod-io/gitpod/pull/11119/files#r913151873).